src/Take.Blip.Builder/Models/StateExtentions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Take.Blip.Builder.Models
+{
+    /// <summary>
+    /// Extension methods of <see cref="State"/>
+    /// </summary>
+    public static class StateExtentions
+    {
+        public static bool IsNotNullAndInputExpirationTimeEnabled(this State state)
```

Comment:
I think this got much better, but maybe the name is too long now... The name doesn't explain every detail of what is being done. The fact that the state and its input should not be null is implicit anyway.

I think just `InputExpirationTimeEnabled` is enough. This makes sense for you?

Reply:
It makes sense. Method name tries to explain what it does, but `InputExpirationTimeEnabled` is what really matters.